test/assets/unrunnableTestCases.csv

```diff
@@ -3121,145 +3092,46 @@ orderBy63,Error: Not Implemented: default namespace element.
+orderbywithout-40,AssertionError: Expected XPath <results> { for $x in (-0.000000000000000001,-0.00000000000000001,-0.0000000000000001,-0.000000000000001,-0.00000000000001,-0.0000000000001,-0.000000000001,-0.00000000001,-0.0000000001,-0.000000001,-0.00000001,-0.0000001,-0.000001,-0.00001,-0.0001,-0.001,-0.01,-0.0,-0.1) order by string($x) return string($x) } </results> to resolve to the given XML. Expected <results>-0.000001 -0.00001 -0.0001 -0.001 -0.01 -0.1 -1E-10 -1E-11 -1E-12 -1E-13 -1E-14 -1E-15 -1E-16 -1E-17 -1E-18 -1E-7 -1E-8 -1E-9 0</results> to equal <results>-0.000000000000000001 -0.00000000000000001 -0.0000000000000001 -0.000000000000001 -0.00000000000001 -0.0000000000001 -0.000000000001 -0.00000000001 -0.0000000001 -0.000000001 -0.00000001 -0.0000001 -0.000001 -0.00001 -0.0001 -0.001 -0.01 -0.1 0</results>
 K2-OrderbyExprWithout-1,Error: XPST0003: Unable to parse: "declare base-uri "http://www.w3.org/2005/xpath-functions/"; let $i as xs:integer* := (1, 2, 3) order by 1 collation "collation/codepoint" return $i". Expected "'" or "\"" but " " found. declare base-uri "http://www.w3.org/2005/xpath-functions/"; let $i as xs:integer* := (1, 2, 3) order by 1 collation[Error is around here] "collation/codepoint" return $i
 K2-OrderbyExprWithout-2,AssertionError: expected [Function] to throw error matching /XQST0076/ but got 'XPST0003: Unable to parse: "declare base-uri "http://www.w3.org/2005/xpath-functions/"; let $i as xs:integer* := (1, 2, 3) order by 1 collation "collation/" return $i".\nExpected "\'" or "\\"" but " " found.\ndeclare base-uri "http://www.w3.org/2005/xpath-functions/"; let $i as xs:integer* := (1, 2, 3) order by 1 collation[Error is around here] "collation/" return $i'
 K2-OrderbyExprWithout-3,AssertionError: expected [Function] to throw error matching /XQST0076/ but got 'XPST0003: Unable to parse: "declare base-uri "http://www.w3.org/2005/xpath-functions/"; let $i as xs:integer* := (1, 2, 3) order by 1 collation "collation/" return $i".\nExpected "\'" or "\\"" but " " found.\ndeclare base-uri "http://www.w3.org/2005/xpath-functions/"; let $i as xs:integer* := (1, 2, 3) order by 1 collation[Error is around here] "collation/" return $i'
-K2-OrderbyExprWithout-5,Error: Not implemented: orderByClause is not implemented yet.
```

Comment:
`K2-OrderbyExprWithout-46,Error: XPTY0004: Could not cast values` is something we'll need to look at